root/src/main/java/com/example/servingwebcontent/MovieListController.java

```diff
@@ -23,4 +29,28 @@ public String login() {
 	public String register() {
 		return "register";
 	}
+
+	@GetMapping("/test")
+	public String test() throws java.io.FileNotFoundException {
+		BufferedReader reader;
+		String user = "", pass = "";
+	
+		try {
+			reader = new BufferedReader (new FileReader("./userInfo.txt"));
+			String line = reader.readLine();
+			user = line;
+			line = reader.readLine();
+			pass = line;
+			reader.close();
+		} catch (IOException e) { System.out.println(e); }
```

Comment:
Still has a println and no other exception here. Something like an IllegalStateException may be good here.